mdp/signal_node.py

```diff
@@ -187,7 +174,14 @@ def _function_infodict(func):
             argnames.append(varargs)
         if varkwargs:
             argnames.append(varkwargs)
-        signature = _legacysignature(inspect.signature(func))
+        try:
+            signature = _legacysignature(inspect.signature(func))
+        except:
```

Comment:
Do we know the raised exception more specifically? I think a plain `except` should be avoided if reasonably possible.

Reply:
Yes, you are right. It should be an `AttributeError`. See [build](https://travis-ci.org/mdp-toolkit/mdp-toolkit/jobs/625163939#L4622).